lib/rouge/lexers/bpf.rb

```diff
@@ -13,25 +13,39 @@ class BPF < RegexLexer
       ).join('|')
 
       MISC_KEYWORDS = %w(
-        be16 be32 be64 exit lock
+        be16 be32 be64 exit lock map
       ).join('|')
 
       state :root do
+        # Line numbers and hexadecimal output from bpftool/objdump
+        rule %r/(\d+)(:)(\s+)(\(\h{2}\))/i do
+          groups Generic::Lineno, Punctuation, Text::Whitespace, Generic
+        end
+        rule %r/(\d+)(:)(\s+)((\h{2} ){8})/i do
+          groups Generic::Lineno, Punctuation, Text::Whitespace, Generic
```

Comment:
```suggestion
          groups Generic::Lineno, Punctuation, Text::Whitespace, Literal::Number::Hex
```

Same issue as above.

Reply:
See above comments.